src/components/JobDescriptionModal/index.jsx

```diff
@@ -3,83 +3,95 @@ import Salary from "../Salary/index.jsx";
-                                    Salary:
-                                </div>
-                                <div className={'col'}>
-                                    Type:
-                                </div>
-                                <div className={'col'}>
-                                    Date posted:
-                                </div>
-                                <div className={'col'}>
-                                    Skills:
-                                </div>
+                        <div className={'modal-body'}>
+                            <div className={'container d-flex p-4 gap-2 align-items-center bg-primary mb-4'}>
+                                <p className={'text-bg-primary m-0'}>{jobInfo.job_title}</p>
+                                {jobInfo.type ? <TypeBadge jobType={jobInfo.type} bgColor={'bg-success'}/> : null}
+                            </div>
+                            <div className={'container d-flex m-1 align-items-center'}>
+                                <img className={'float-start img-responsive me-1'} width={40} height={40}
+                                     src={jobInfo.logo}/>
+                                <p className={'m-0'}>{jobInfo.company}</p>
                             </div>
 
-                            <div className={'row'}>
-                                <div className={'col'}>
-                                    <Salary salary={jobInfo.salary}/>
-                                </div>
-                                <div className={'col'}>
-                                    <TypeBadge jobType={jobInfo.type}/>
+                            <div className={'container m-1'}>
+                                <div className={'row'}>
+                                    <h5>Key facts:</h5>
                                 </div>
-                                <div className={'col'}>
-                                    {jobInfo.posted}
+                                <div className={'row'}>
+                                    <div className={'col-3'}>
+                                        Salary:
+                                    </div>
+                                    <div className={'col-3'}>
+                                        Type:
+                                    </div>
+                                    <div className={'col-3'}>
+                                        Date posted:
+                                    </div>
+                                    <div className={'col-3'}>
+                                        Skills:
+                                    </div>
                                 </div>
-                                <div className={'col'}>
 
-                                    {/*{*/}
+                                <div className={'row'}>
+                                    <div className={'col-3'}>
+                                        <Salary salary={jobInfo.salary}/>
```

Comment:
Some of the job listings don't have a salary. For Story 1, we have shown these as 'competitive'.